.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @sweetpea22 @web3/web3-js
```

Comment:
```suggestion
* @web3/web3-js
```
@sweetpea22  is not in our org?

Reply:
That's Cindy, added her as outside collaborator on this repo^^